middle_end/flambda2/terms/static_const.ml

```diff
@@ -189,15 +189,15 @@ let [@ocamlformat "disable"] print ppf t =
       Flambda_colours.pop
       (Format.pp_print_list
         ~pp_sep:(fun ppf () -> Format.pp_print_string ppf "@; ")
-        (Or_variable.print (fun ppf n -> Format.fprintf ppf "%ld" n)))
+        (Or_variable.print Numeric_types.Int32.print))
```

Comment:
I don't think this file needed to be modified, but the patch looks good so it's fine.